### DISCUSSION TITLE
How to assign records to group by default ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#16500] [How to] [answered]
Hi,

I want to assign new records to group by default for some users.
I did what is recommended in the following issue #11097 but it looks like it's not working.
All newly created records remains assigned to the user.
Is it still the right way to auto assign records to group instead of users ?
Does anybody know the steps to configure auto assign records to group by default ?
Regards,

**Comment by apcloic** [comment]
Hi,

So should I understand that this no longer works ?

User's role :
I did set the value to: from panel
Then set the group in the panel.

Then in RecordAllocation module (assign owner), I select the module and the user, then I drag and drop user's group from right to left panel.

When creating new record, it's still assign to user and not user's group.
Before saving the new record, I've noticed that assigned to field value appears as "undefined" and while shared with users is empty.

![how_to_assign_to_group_by_default](https://user-images.githubusercontent.com/2930560/192598612-ef29aa8b-ce81-4498-aa7d-651785040119.gif)

**Comment by rskrzypczak** [comment, marked as answer]
- assign modules to groups.
![image](https://user-images.githubusercontent.com/10221999/192691891-06dc4b57-746f-4482-85a3-68ace8452cdf.png)
- fixed here: https://github.com/YetiForceCompany/YetiForceCRM/commit/dd70ba7c1520c28c5b8a482255e1a112b394ea73, you have to manually move it to your system if you want this functionality to work for the groups themselves.
- besides, admin is not a good user for verification.

**Reply by apcloic** [reply]
@rskrzypczak Thank you very much, works perfectly.
